docs/options.md

```diff
@@ -344,3 +344,7 @@ with info about your project (name and website) so we can add an entry for you.
 1. mypy-protobuf
    * Website: https://github.com/nipunn1313/mypy-protobuf
    * Extension: 1151-1154
+
+1. trpc-options
+   * Website: ???
```

Comment:
We require some linkage to an external website for our documentation. What is TRPC?

Reply:
TRPC is a Remote Proceduce Call framework waiting to be open-sourced. In this case, we haven't have a public website url. Is this ok for us to reserve thses option numbers?

Reply:
We require you have a public website before adding the option number. Update this when you have one.